runbotics/runbotics-orchestrator-ui/src/src-landing/translations/index.ts

```diff
@@ -0,0 +1,17 @@
+import enBenefitsTranslations from '#src-landing/translations/en/benefits.json';
+import enHeroTranslations from '#src-landing/translations/en/hero.json';
+import enIntegrationTranslations from '#src-landing/translations/en/integration.json';
+import plBenefitsTranslations from '#src-landing/translations/pl/benefits.json';
+import plHeroTranslations from '#src-landing/translations/pl/hero.json';
+import plIntegrationTranslations from '#src-landing/translations/pl/integration.json';
+
+const landingPageTranslations = {
+    ...enHeroTranslations,
```

Comment:
do not mix `en` and `pl` translations see how it is done with regular translations, there are separate objects

Reply:
because of this you end up with a ui build error

![image](https://user-images.githubusercontent.com/44121335/207319708-89ea4571-b259-4f4c-a09a-28ec9bd8b946.png)

Reply:
done, i see it now, thanks